great_expectations/data_asset/data_asset.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, *args, **kwargs):
 
         if profiler is not None:
             profiler.profile(self)
+        if data_context and hasattr(data_context, '_expectation_explorer_manager'):
```

Comment:
Not a blocker for this review, but I think we should (eventually) look at how to refactor the DataAsset class without it having to know about things like profilers and the data_context. I suspect this is something that'll fall out of the (future) refactor to `BasicValidationOperators`.

Pinging @jcampbell to share state.